### PR TITLE
Restructuring OpenShift templates documentation

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

**Comment by sterobin** [comment]
@mramendi , Great job here. One suggestion: See the screenshot below of the rendered output:

image

I would remove II Deploying RHPAM authoring env ..., 5. Overview ..., 6. Preparing. Chaps. 5 and 6 are already covered, and you don't need to segregate the new assembly for authoring env like this because the objective of how you've set up is to say "Deploying RHPAM environments ...", and here they are. Then you list each type of environment and it's sub-sections, as you've done. So can just go from 4. Migrating ... to 5. Trial environment to 6. Authoring environment and then it's all consistenct. Regarding the Trial environment, I'd recommend structuring it similarly to the other sections you have, where it goes "Trial environment" concept like you do with the others, followed by "Deploying a trial environment ..." within it.

So with this idea, you'd end up with a TOC like this:

I. Deploying a Red Hat Process Automation Manager environment on Red Hat OpenShift Container Platform using Operators
1. Overview of Red Hat Process Automation Manager on Red Hat OpenShift Container Platform
2. Preparing to deploy Red Hat Process Automation Manager in your OpenShift environment
3. Deployment and management of a Red Hat Process Automation Manager environment using OpenShift operators
4. Migrating information from a deployment on Red Hat OpenShift Container Platform version 3
**5. Trial environment**
**5.1. Deploying a trial environment**
6. Authoring environment
9. Environment with immutable servers
... etc.

Wdyt?

**Comment by sterobin** [comment]
@mramendi, per our chat convo, thanks for the fix, and makes sense. 

I still think would be nice if it were this:

Trial environment (con)
  Deploying a trial environment (proc)

Just for the sake of consistency with how you've structured the others. But not at all critical. Just a nice-to-have, and a subjective one. So just food for your thought, otherwise good on my end either way.

File: doc-content/enterprise-only/openshift/ba-openshift-overview-con.adoc [hunk 1]
```diff
@@ -42,49 +42,28 @@ endif::PAM[]
...
 You can arrange these and other components into various environment configurations within OpenShift.
 
-ifeval::["{context}"=="openshift-freeform"]
-endif::[]
-
-ifeval::["{context}"!="openshift-operator"]
+ifeval::["{context}"=="openshift-templates"]
 The following environment types are typical:
 
+* _Trial_: an environmend for demonstration and evaluation of {PRODUCT}. This environment includes {CENTRAL} and a {KIE_SERVER}. You can set it up quickly and use it to evaluate or demonstrate developing and running assets. However, the environment does not use any persistent storage and any work you do in the environment is not saved.
```
Comment (heathjoy) [inline doc-content/enterprise-only/openshift/ba-openshift-overview-con.adoc:48]: typo: environmend

File: doc-content/enterprise-only/openshift/ba-openshift-overview-con.adoc [hunk 2]
```diff
@@ -42,49 +42,28 @@ endif::PAM[]
...
-ifeval::["{context}"!="openshift-ansible-playbook"]
-For instructions about deploying this environment, see {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-authoring[_{DEPLOYING_AUTHORING_ON_OPENSHIFT}_].
-endif::[]
 * _Managed deployment_: An environment for running existing services for staging and production purposes. This environment includes several groups of {KIE_SERVER} pods; you can deploy and undeploy services on every such group and also scale the group up or down as necessary. Use {CENTRAL} Monitoring to deploy, run, and stop the services and to monitor their execution.
-ifeval::["{context}"!="openshift-ansible-playbook"]
 +
-You can deploy two types of managed environment. In a _freeform_ server environment, you initially deploy {CENTRAL} Monitoring and one {KIE_SERVER}. You can additionally deploy any number of {KIE_SERVERS}. {CENTRAL} Monitoring can connects to all servers in the same namespace. For instructions about deploying this environment, see {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-freeform[_{DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}_].
```
Comment (heathjoy) [inline doc-content/enterprise-only/openshift/ba-openshift-overview-con.adoc:59]: typo: Monitoring can connects to all servers in the same namespace.

File: assemblies/assembly-openshift-operator.adoc [hunk 3]
```diff
@@ -14,7 +21,9 @@ As a system engineer, you can deploy a {PRODUCT} environment on {OPENSHIFT} to p
 * You are logged in to the project using the OpenShift web console.
```
Comment (heathjoy) [inline assemblies/assembly-openshift-operator.adoc:21]: should be "into"

**Pull request merged**